docker/hadoop/cdh/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # pull base image
-FROM ubuntu:16.04
+FROM harbor.h2o.ai/opsh2oai/h2o-3/dev-r-3.4.1-jdk-8:10
```

Comment:
Using this base image, might be a problem. Sparkling Water images are based on these images, and some of them need Java 7. What is the reason to use the R image as base please?

Reply:
well we need R, first I tried copy&pasting the code to install R but 
- image build failed because I reached CRAN API limits 
- building the images took forever, this speeds it up a lot
- I dont understand the note about java7, we are using a specific java8+R341 image
- less code/reuse = win